### ISSUE TITLE
Repeater - Required fields failing

### ISSUE THREAD

**Issue by Timusan** [verbb/formie#485]
**Description**

Required fields inside a repeater fail validation, even when filled in.
The error message that comes back says that the required fields cannot be empty even though they are filled in.

**Steps to reproduce**
1. Create a new, empty form
2. Add a Repeater
3. Drag a single line text field (or any other field) into the repeater (my use case it happened with both a date field and a single line text field)
4. Set the field as required
5. Fill in the field
6. Now try to submit -> Validation errors occur, even though the field is filled in

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.5
- Craft version: 3.6.15
- Multi-site: Yes or No

**Comment by Timusan** [comment]
Upon further experimenting: when I set the fields as not required, fill them in and submit the form they are empty on the resulting submission. So the validation message is "correct", the content of the fields is simply not stored when used in a repeater.

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.5",
  "...": "..."
}
```

Then run `composer update`.

**Comment by Timusan** [comment]
Unfortunately, while the validation now passes for required fields, the submission is still left empty.

Edit: I should point out: the fields in the repeater are empty (as in, only the "+ add more" button is shown, no entries in the repeater), other fields are stored just fine.

**Comment by engram-design** [comment]
Should be fixed in the `dev-craft-3` branch.

**Comment by Timusan** [comment]
The repeater now saves ... but only the first entry :sweat_smile: .

**Comment by Timusan** [comment]
Nope, scratch that last comment, this was a composer issue ... it works!

Thanks @engram-design for the fast action on this! :partying_face:

**Comment by engram-design** [comment]
Fixed in [1.4.6](https://github.com/verbb/formie/releases/tag/1.4.6)
